include/pressio/ops/eigen/ops_level2.hpp

```diff
@@ -55,6 +55,12 @@ namespace pressio{ namespace ops{
  * y = beta * y + alpha*op(A)*x
 */
 
+// Implementation notes.
+// Eigen requires that:
+// - scalar type of A, x and y is the same
+// - alpha and beta types are same as scalar type
+// - scalar can be constructed from double, eg. Scalar(1)
```

Comment:
where is "scalar can be constructed from double, eg. Scalar(1)" coming from?

Reply:
also we need to be careful here, becuase the requirement that eigen sets are NOT necessarily what we need to set here.

Reply:
@fnrizzi 
> where is "scalar can be constructed from double, eg. Scalar(1)" coming from?

please see [#446 Eigen note](https://github.com/Pressio/pressio/issues/446#issuecomment-1297773123)